004-governance.Rmd

```diff
@@ -0,0 +1,153 @@
+The only requirement is to follow the code of conduct.
+
+Packages don't maintain an explicit list of contributors but acknowledge them in blog posts using data from GitHub aggregated by `usethis::use_tidy_thanks()`.
+
+### Authors
+
+Contributors who have made significant and sustained contributions can be invited to become authors.
+Authors are collectively responsible for day-to-day development of the package, including responding to issues and reviewing pull requests.
+An author possesses two special powers:
+
+-   They have **write** access on GitHub so they can triage issues, request review on PRs, and merge them.
+
+-   They are listed in `Authors@R` so they receive credit when others cite the package.
+
+Authors are expected to follow our standard processes, such as:
+
+-   **Code contribution**: code is usually contributed via PR, even for authors who could push directly.
+    Particularly high-stakes project may want to protect the main branch and require "request reviews before merging".
+
+-   **Communication**: authors are involved in most of the interactions with contributors and thus need to set a welcoming and inclusive tone for the project.
+
+-   **PR review**: all pull requests should be reviewed by at least one other author.
+    In general, there is no expectation that PRs contain clean commit histories, but it's appreciated where possible.
+
+-   **Backward compatibility**: any backward incompatible changes (i.e. changes that cause reverse dependencies to fail `R CMD check` or are likely to cause problems in user code) must be approved by the maintainer.
+    Significant backward incompatible changes need to be accompanied with a plan for how they will be communicated to the community.
+
+-   **CRAN releases**: package releases are made on an as-needed basis, and increment either the major, minor, or patch version depending on the scope of the release.
+    The process itself is defined by `usethis::use_release_issue()`.
+
+-   **Decision making:** when a package has multiple authors, where possible decisions are made using rough consensus amongst the authors.
+    If consensus is hard to reach or taking too long, the maintainer will make a decision.
+
+(We expect to flesh these processes out in the coming months.)
+
+Authors are recruited from contributors.
+An invitation to join the authors can be extended to anyone who has made significant and sustained contributions, and has acted in accordance with the code of conduct.
+Any existing author can propose a contributor be invited by emailing the maintainer.
+
+### Maintainer
+
+A maintainer is the author with primary responsibility for the project.
+As well as the responsibilities of an author, they also:
+
+-   Set and clearly communicates the strategic objectives of the project.
+-   Oversee CRAN releases.
+-   On-board new authors.
+-   Mediate conflicts amongst authors.
```

Comment:
A sub-bullet of this is knowing when to use GitHub's built in "lock conversations" features, either temporarily or permanently, when arguments on an issue get out of hand

Reply:
Good idea, I'll think about the wording.

Reply:
Tweaked above bullet to "De-escalate conflict amongst authors and/or users.". Hopefully that captures the spirit, without getting too caught up in the details.